docs/application/dotnet/guides/nui/nui-components/RadioButton.md

```diff
@@ -41,17 +41,17 @@ Following output is generated when a radio button is created using style:
 ![RadioButton2](./media/RadioButton2.png)
 
 
-## Responding to ClickEvent
+## Responding to Clicked
 When you click a radio button, the radio button instance receives a click event.
 You can declare the click event handler as follows:
```

Comment:
Suggestion. How about changing "click" to "clicked"?

```suggestion
When you click a radio button, the radio button instance receives a clicked event.
You can declare the clicked event handler as follows:
```

Reply:
Thank you for your suggestion.
I have updated the patch as you suggested.